testsuite/pom.xml

```diff
@@ -51,6 +51,7 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Why groupId is needed?
P.S there is one maven-surefire-plugin that is missing groupId as well.

Reply:
Well is not "needed" in a theoretical point of view, because in the maven XML schemas ([XSD](https://maven.apache.org/xsd/maven-4.0.0.xsd)) the groupId has `org.apache.maven.plugins` as a default value, so I guess that Maven will handler this XMLs through Jaxb or some other xsd libs, and if the group ID is not given then the default value will be used. But I think that is clear, just write your groupId in all the cases.

Reply:
Look, even on the plugin example, they do it "well":
https://maven.apache.org/plugins/maven-compiler-plugin/plugin-info.html
https://maven.apache.org/surefire/maven-surefire-plugin/usage.html